README.md

```diff
@@ -31,72 +14,74 @@ asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+`node-build` already has a [handful of settings](https://github.com/nodenv/node-build#custom-build-configuration), in additional to that `asdf-nodejs` has a few extra configuration variables:
 
-```
-legacy_version_file = yes
-```
+- `ASDF_NODEJS_NODEBUILD_HOME`: Home for the node-build installation, defaults to `$ASDF_DIR/plugins/nodejs/.node-build`, you can install it in another place or share it with your system
+- `ASDF_NODEJS_NODEBUILD`: Path to the node-build executable, defaults to `$NODE_BUILD_MIRROR_URL/bin/node-build`
+- `ASDF_NODEJS_CONCURRENCY`: How many jobs should be used in compilation. Defaults to half the computer cores
+- `ASDF_NODEJS_VERBOSE_INSTALL`: Enables verbose output for downloading and building. Any value different from empty is treated as enabled.
+- `NODEJS_ORG_MIRROR`: (Legacy) overrides the default mirror used for downloading the distibutions, alternative to the `NODE_BUILD_MIRROR_URL` node-build env var
 
-## Default npm Packages
+### Integrity/signature check
+
+In the past `asdf-nodejs` checked for signatures and integrity by querying live keyservers. `node-build`, on the other hand, checks integrity by precomputing checksums ahead of time and versioning them together with the instructions for building them, making the process a lot more streamlined.
+
+### `.nvmrc` and `.node-version` support
 
-asdf-nodejs can automatically install a set of default set of npm package right after installing a Node.js version. To enable this feature, provide a `$HOME/.default-npm-packages` file that lists one package per line, for example:
+asdf uses a `.tool-versions` file for auto-switching between software versions. To ease migration, you can have it read an existing `.nvmrc` or `.node-version` file to find out what version of Node.js should be used. To do this, add the following to `$HOME/.asdfrc`:
 
 ```
-lodash
-request
-express
+legacy_version_file = yes
 ```
 
-You can specify a non-default location of this file by setting a `ASDF_NPM_DEFAULT_PACKAGES_FILE` variable.
+### Updating node-build definitions
 
-## Problems with OpenPGP signatures in older versions
+Every new node version needs to have a definition file in the `node-build` repository. Because of that, `asdf-nodejs` tries to update the local `node-build` repository at every install command to download definitions for newly added versions.
 
-The plugin automatically imports the NodeJS release team's OpenPGP keys. If you are trying to install a previous release and facing any issue about verification, import the Node.js previous release team's OpenPGP keys to main keyring:
+We provide a command for manually updating `node-build` at your own time:
 
 ```bash
-bash -c '${ASDF_DATA_DIR:=$HOME/.asdf}/plugins/nodejs/bin/import-previous-release-team-keyring'
+asdf nodejs update-nodebuild
```

Comment:
nitpick: Is this command really necessary? If this runs before every install command, why would a user need to run it manually?

Reply:
It's just seemed like a useful command in general, for scripting or just a shortcut before running commands directly on the internal `nodebuild` command via `asdf nodejs nodebuild ...`

Reply:
Why not just update `node-build` when the user invokes `asdf nodejs nodebuild ...`?

Reply:
> Why not just update `node-build` when the user invokes `asdf nodejs nodebuild ...`?

`asdf nodejs nodebuild ...` is just a peek at the current `node-build` installation. I don't think we should forcefully update it on the command invocation

> The presence of this command may confuse users.

I agree with you. Maybe we should make it more clear that `asdf` is already auto-updating `node-build` on installations and that running `asdf nodejs update-nodebuild` is optional.

Reply:
I did some changes to the documentation, check if that is better